versions.yaml

```diff
@@ -177,7 +177,7 @@ externals:
   attestation-agent:
     description: "Provide attested key unwrapping for image decryption"
     url: "https://github.com/confidential-containers/attestation-agent"
-    branch: "main"
+    tag: "v0.1.0"
```

Comment:
I think the model from the other components in versions.yaml is to call it `versions:` and then it can be a release, or commit. As I mentioned in slack, I think ‘pinning’ the attestation-agent rather than just pulling `main` is a good  start, but my concern about just using a tag is when would the tag be created & bumped? If it isn’t until the release we won’t be able to do any integration tests of new features, whereas using a commit would let us pull in a version of our choice?

For example right now Ding has added cosign support into attestation-agent, which we'd ideally test with kata as soon as possible, but it isn't in the tagged version. Instead we can pull in the commit (which would trigger the kata-payload as a bonus) and check that it works, then that test would like us judge whether that commit is eligible to be tagged for the next release. As we get to the release date, then tagging a commit and swapping the version.yaml reference to the tag sounds fine, but after code freeze ends we'd probably want to start picking up new features again.
Does this make sense?